pandas/tests/reshape/test_pivot.py

```diff
@@ -2670,10 +2668,10 @@ def test_pivot_columns_not_given(self):
 
     # this still fails because columns=None gets passed down to unstack as level=None
     # while at that point None was converted to NaN
-    @pytest.mark.xfail(
-        using_string_dtype(), reason="TODO(infer_string) None is cast to NaN"
-    )
-    def test_pivot_columns_is_none(self):
+    # @pytest.mark.xfail(
+    #     using_string_dtype(), reason="TODO(infer_string) None is cast to NaN"
+    # )
```

Comment:
This can be removed instead of put in comments?

Reply:
I must of had the wrong branch checked out when making this. Will cleanup. This should just have the whatsnew note.

Reply:
Ready now.